src/features/languageStatus.ts

```diff
@@ -19,6 +27,15 @@ export class LanguageStatusFeature implements StaticFeature {
     private outputChannel: vscode.OutputChannel,
   ) {}
 
+  fillInitializeParams?: ((params: InitializeParams) => void) | undefined;
+
+  preInitialize?:
+    | ((capabilities: ServerCapabilities<any>, documentSelector: DocumentSelector | undefined) => void)
+    | undefined;
```

Comment:
Do we need those two optional method definitions or can they (and the additional imports) be removed?

Reply:
This applies to our other static features as well

Reply:
I implemented the interface figuring it was no harm since I had to add `clear` anyway. Removed.